spec/features/admin/index_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
```

Comment:
Its looking like our '/admin' path will be an alias for the admin dashboard?

Reply:
Good commits! Ready to merge

Reply:
yeah I changed the route because I realized the user story was 
```
As an admin,
When I visit the admin dashboard (/admin)
Then I see a header indicating that I am on the admin dashboard
```
so I decided that we don't actually need admin/dashboard link.

Reply:
Nailed it!